contracts/libraries/BitMath.sol

```diff
@@ -11,37 +11,17 @@ library BitMath {
     /// @param x the value for which to compute the most significant bit, must be greater than 0
     /// @return r the index of the most significant bit
     function mostSignificantBit(uint256 x) internal pure returns (uint8 r) {
-        require(x > 0);
+        require(x > 0, "Value must be greater than 0");
```

Comment:
Adding this 'statement' inside the `require` block usually costs gas. So there might be a chance that the community might ask you to revert this kind of change.
However, the code readability changes are appreciated!